std/ws/mod.ts

```diff
@@ -327,6 +329,26 @@ class WebSocketImpl implements WebSocket {
     return this.enqueue(frame);
   }
 
+  async write(p: Uint8Array): Promise<number> {
+    await this.send(p);
+
+    return p.byteLength;
+  }
+
+  async read(p: Uint8Array): Promise<number | null> {
+    for await (const ev of this.receive()) {
+      if (ev instanceof Uint8Array) {
+        return copyBytes(p, ev);
```

Comment:
`copyBytes` ignores overflowed data of src. It seems to discard data in frames, doesn't it?.

Reply:
I think you are right. Would something like this make sense instead?

```typescript
private readBacklog: Uint8Array = new Uint8Array()

  async read(p: Uint8Array): Promise<number | null> {
    for await (let ev of this) {
      if (ev instanceof Uint8Array) {
        if (this.readBacklog.byteLength) {
          const copiedFromBacklog = copyBytes(this.readBacklog, p);
          this.readBacklog = this.readBacklog.subarray(copiedFromBacklog);
          
          return copiedFromBacklog + copyBytes(ev, p, copiedFromBacklog);
        }

        const copied = copyBytes(ev, p);
        this.readBacklog = ev.subarray(copied);

        return copied;
      }

      if (typeof ev === "string") {
        return copyBytes(encode(ev), p);
      }

      // if (typeof ev === "string") {
    }

    return null;
  }
```